manifests/fastd.pp

```diff
@@ -26,6 +28,15 @@
       ensure => file,
              notify => Service[ffnord::resources::fastd],
              content => template('ffnord/etc/fastd/fastd.conf.erb');
+    "/etc/fastd/${mesh_code}-mesh-vpn/fastd-blacklist.sh":
+      ensure => file,
+             notify => Service[ffnord::resources::fastd],
+             content => template('ffnord/etc/fastd/fastd-blacklist.sh.erb'),
+             mode => '0766';
+   "/etc/fastd/${mesh_code}-mesh-vpn/fastd-blacklist.json":
```

Comment:
thanks for pointing that out ;) now that I had a closer look at the fastdbl stuff, I understand what you are trying to achieve here.

please understand my initial comment like this: there is no puppet support for managing the fastd-blacklist.json file. please implement such a thing in a way where we can choose between using the traditional approach with the fastd key repo and the blacklist approach. the latter should then include something to automatically pull the blacklist file from a remote repo (as shown in https://github.com/ffruhr/fastdbl)